lib/VeresOneClient.js

```diff
@@ -46,9 +45,9 @@ class VeresOneClient {
    * Fetches a DID Document for a given DID. If it contains a #hash fragment,
    * it's likely a key id, so just return the subgraph, not the full doc.
    *
-   * @param did {string} DID uri (possibly with hash fragment)
+   * @param did - {string} DID uri (possibly with hash fragment).
```

Comment:
type `{string}` should precede the name `did` correct?  Here and throughout.  I thought the jsdoc linter was enforcing this?

Reply:
it should be {Type} paramater_name - description good catch.